web-console/src/app/(root)/(blank)/auth/aws/page.tsx

```diff
@@ -19,7 +20,12 @@ export default () => {
     setAuth({
       bearer: params['access_token'],
       user: {
-        username: 'anonymous'
+        avatar: jwtIdPayload['picture'],
```

Comment:
What happens if the JWT payload does not have a picture or these other claims (like phone number...)?

Reply:
The email / phone are just not shown, and there is just generic icon of a profile image in place of an avatar